core/templates/pages/skill-editor-page/editor-tab/skill-concept-card-editor/skill-concept-card-editor.component.ts

```diff
@@ -88,6 +88,9 @@ angular.module('oppia').component('skillConceptCardEditor', {
       $scope.onSaveExplanation = function(explanationObject) {
         SkillUpdateService.setConceptCardExplanation(
           $scope.skill, explanationObject);
+        // TODO(#8521): Remove the use of $rootScope.$apply()
+        // once the controller is migrated to angular.
+        $rootScope.$apply();
         initBindableFieldsDict();
```

Comment:
This is needed because the change count in the navbar is not getting updated, as change detection is not working here due to hybrid mode. So we have to apply it manually.